src/api/parser/readme.md

```diff
@@ -0,0 +1,30 @@
+# Image Service
+
+The Parser service parses posts from user's feeds.
+
+## Install
+
+```
+npm install
+```
+
+## Usage
+
+```
+# normal mode
+npm start
```

Comment:
These don't exist, let's make sure this file represents what's really happening.